templates/definition/charger/viessmann.yaml

```diff
@@ -140,13 +140,13 @@ params:
     default: 0
   - name: target_temperature
     description:
-      de: Zieltemperatur für Einmal-Warmwasser-Zubereitung
-      en: Target temperature for one-time charge
+      de: Veraltet - Zieltemperatur wird nicht mehr benutzt
+      en: Outdated - target temperature not used any longer
```

Comment:
Leave text as-is, add

    deprecated: true

Reply:
done

<img width="757" height="232" alt="image" src="https://github.com/user-attachments/assets/220d7239-fe72-477b-94fb-14bcd90405b4" />